blackmarket/UI/Common/SearchBar.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SearchBar.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 12/6/22.
+//
+
+import SwiftUI
+
+struct SearchBar: View {
+    
+    @Binding var text: String
+    @State private var isEditing = false
+ 
+    var body: some View {
+        ZStack(alignment: .trailing) {
```

Comment:
why is a ZStack used here? Shouldn't the magnifying glass icon be outside of the field? Otherwise, you should consider that the text could overlap with the image.

Reply:
I fixed this to avoid overlaping

Reply:
But why is a ZStack used if you don't need one element over the other one?

Reply:
You are right let me change this